mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -194,6 +204,22 @@ def __find_pod_id(pods, pod_id):
         return None
 
     @staticmethod
+    def __find_num(pods, pod_num):
+        for pod in pods:
+            if pod.node.attrib['position'] == pod_num:
+                return pod.text
+        return None
+
+    @staticmethod
+    def _find_did_you_mean(res):
+        value = []
+        root = res.tree.find('didyoumeans')
+        if root:
+            for result in root:
+                value.append(result.text)
+        return value
```

Comment:
Assuming we **always** pick the first item from `others` array, it seems to me this method logic should change.
We just need to return the first found alternative value from `root` instead of adding all possible into an array. Then, the `others` array would become something else which represents that alternative value. 
```
    def _find_did_you_mean(res):
        root = res.tree.find('didyoumeans')
        if not root:
            return None
        return root[0].text
```